selfdrive/car/car_helpers.py

```diff
@@ -23,6 +24,8 @@ def get_startup_event(car_recognized, controller_available):
     event = EventName.startupNoCar
   elif car_recognized and not controller_available:
     event = EventName.startupNoControl
+  elif ANDROID and "letv" not in open("/proc/cmdline").read():
```

Comment:
why can't this use not LEON?

Reply:
leon is defined a line above the if in updated, not in the hw abstraction layer. we could define in the hw abstraction layer, but would prefer to keep these temp changes as local as possible